README.md

```diff
@@ -494,6 +494,9 @@ If you are using Supervised Home Assistant (formerly known as the Hassio/Hass.io
 
 [zha-network-card](https://github.com/dmulcahey/zha-network-card) is a custom Lovelace card that displays ZHA network and device information in Home Assistant
 
+## Zigzag
+[Zigzag](https://github.com/Samantha-uk/zigzag-v1) is an custom card/panel for [Home Assistant](https://www.home-assistant.io/) that displays a graphical layout of Zigbee devices and the connections between them. Zigzag can be installed as a panel or a custom card and relies on the data provided by the [zha-map](https://github.com/zha-ng/zha-map) integration commponent.
+
```

Comment:
does this still actually need zha-map?

Reply:
I Don't know and don't think so

Reply:
Removed "and relies on the data provided by the zha-map integration component"